ux/marketing blog post (#64)
Contenda-Team/Contenda-landing-page: src/posts/marketing.md

---
layout: "../layouts/BlogPost.astro"
title: "Finding Harmony in Marketing and UX"
slug: marketing
description: "How a holistic design approach improved Contenda's Marketing and UX efforts simultaneously"
author: "Sally Chen"
added: "May 30 2023"
tags: ["user experience"]
---

> When we think of teamwork in the world of user experience (UX), we often imagine “design and engineering holding hands”. However, the idea of “design and marketing holding hands”is not as common. While marketing aims to persuade users to take action, UX is all about meeting their needs and expectations. It can be challenging to sell and serve simultaneously. Balancing the two can be quite a challenge.

Reviewer: ```suggestion
When we think of teamwork in the world of user experience (UX), we often imagine design and engineering working together. However, the idea of design and marketing working together is not as common. While marketing aims to persuade users to take action, UX is all about meeting their needs and expectations. It can be challenging to balance sell and serve simultaneously. 
```

Reviewer: I love the holding hands part as much as the next one, but I don't think it's a good idea to make that reference because it's an inside joke

> However, as Contenda experienced rapid growth with multiple ongoing releases and partnerships, there came a point when I had to tackle both go-to-market and UX efforts as the sole designer. It was a challenging task, and I initially struggled to find the sweet spot between creating visually appealing designs to attract new users and providing a seamless experience for existing ones.

Reviewer: ```suggestion
However, as Contenda experienced rapid growth with multiple ongoing releases, there came a point when I had to tackle both go-to-market and UX efforts as the _sole designer_. This is my advice to all the solo designers out there trying to juggle a startup's many priorities. 
```

## Embracing a holistic approach

> To overcome these hurdles, I decided to **set aside** the distinction between marketing and UX, and instead focus on the bigger picture. I asked myself questions like, "What do I want people to remember about Contenda?" and "How can I make our platform engaging, informative, and valuable?" 

Reviewer: ```suggestion
To overcome these hurdles, I decided to **set aside** the distinction between marketing and UX, and instead focus on the bigger picture. I asked myself questions like:

- "What do I want people to remember about Contenda?" 
- "How can I make our platform engaging, informative, and valuable?" 
```

> While seeking answers to these questions, I found that one of Contenda’s biggest problems was the lack of consistency. Take a look below - some pages looked like they were still under development, while others exuded a polished, professional vibe. The colors, typography, and layout were completely different too. This inconsistency was exacerbated by the numerous channels through which others discovered Contenda, such as emails, social media posts, and partnership pages. It was crucial to convey Contenda's value consistently across these touch-points.

Reviewer: ```suggestion
While seeking answers to these questions, I found that one of Contenda’s biggest problems was the lack of consistency. Take a look below - some pages looked like they were still under development, while others exuded a polished, professional vibe. The colors, typography, and layout were completely different too. This inconsistency was exacerbated by the numerous customer channels ranging from emails to social media to partnership pages. It was crucial to convey Contenda's value consistently across these touch-points.
```

Throughout this journey to harmonize marketing and UX, I also discovered several super helpful and handy design practices:

1. **Create reusable components** - I transformed most visual assets into reusable components across the entire Contenda ecosystem. For instance, when updating our social media avatars, I noticed that adding a drop shadow to the logo instantly made it more visually appealing and helped it stand out against the dark background. This inspired me to apply the same shadow in other areas with dark backgrounds, like our hero section and soon-to-come new sign-up page!
![Drop shadows across our landing page and soon to come new sign-up page](/blogimages/dropshadow.png)

2. **Build templates for everything** - With a multitude of changes and rapid releases, it’s essential to keep designs flexible yet scalable. Templates, like the branding-related ones I made in Figma, proved to be incredibly useful. Whenever we established a new partnership, someone else could simply drop the logo and export the file to be used in the relevant social media posts. Skeletal layouts for signup/submission forms were also beneficial for experimenting with new designs.

> 3. **Establish a central hub for visual documentation** - This was particularly critical for both the marketing and front-end sides to maintain visual consistency. I found tools like [zeroheight](https://zeroheight.com/) extremely helpful for easily pushing changes and allowing the entire team and the public to view our brand guidelines.

Reviewer: You should ask Zeroheight to do comarketing with us!!!
